azure/dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.6-alpine
+
+LABEL maintainer="Maxime Sibellas <maxime.sibellas@ekino.com>"
+
+ARG TERRAFORM_VERSION
+ARG AZURE_CLI
+
+RUN apk update \
+    && apk add --update -q --no-cache bash ca-certificates unzip wget \
+    && pip install --upgrade pip awscli \
+    && wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip -O /tmp/terraform.zip \
+    && cd /tmp \
+    && unzip -o terraform.zip \
+    && cd - \
+    && mv /tmp/terraform /usr/bin/ \
+    && chmod +x /usr/bin/terraform \
+    && apk add build-base libffi-dev openssl-dev \
```

Comment:
maybe you could group all packages installations

Reply:
done for both apk & pip :)